### PR TITLE
MFW-3684: added logging schema

### PR DESCRIPTION
**Pull request by yogesh-chhetri**
JIRA:
https://awakesecurity.atlassian.net/browse/MFW-3684

### CODE REVIEW

File: v1/system/system_schema.json [hunk 1]
```diff
@@ -13,7 +13,38 @@
                 "timeZone": { "$ref": "#/definitions/time_zone_settings" },
                 "setupWizard": { "$ref": "#/definitions/setup_wizard_settings" },
                 "autoUpgrade": { "$ref": "#/definitions/auto_settings" },
-                "autoBackup": { "$ref": "#/definitions/auto_settings" }
+                "autoBackup": { "$ref": "#/definitions/auto_settings" },
+                "logging": { "$ref": "#/definitions/logging" }
+            }
+        },
+        "logging": {
```
Comment (AlistairJohnParker) [inline v1/system/system_schema.json:20]: Should there be a unit test for this ?

Comment (yogesh-chhetri) [reply v1/system/system_schema.json:20]: Hi Al,
I don't see any UTs for [system_schema.json](https://github.com/untangle/mfw_schema/blob/master/v1/system/system_schema.json) (there's no validate*.py in directory =>  `mfw_schema/v1/system/`)

should I add a validate_system_schema.py here ? or is there any other place I need to put the test , can you please suggest ?

Comment (AlistairJohnParker) [reply v1/system/system_schema.json:20]: You can add a task to .vscode/tasks.json, system/test_system_schema.json and systen/validatsystem.py

Comment (yogesh-chhetri) [reply v1/system/system_schema.json:20]: Hi Al,  I have updated the `.vscode/tasks.json` , added `system/test_system_schema.json` and `system/valitate_system_schema.py`

**Review by AlistairJohnParker — APPROVED** [review]
Looks good

**Review by manuwelakanade — APPROVED** [review]

**Pull request merged**